nncf/quantization/algo.py

```diff
@@ -684,6 +682,8 @@ def _get_quantizer_setup(self, target_model: NNCFNetwork) -> SingleConfigQuantiz
                                                                   self._range_init_params,
                                                                   self._debug_interface)
         single_config_quantizer_setup = setup_generator.generate_setup()
+        # TODO: how to initialize in case of loading it from setup_state?
+        #  should be part of setup?
         self._build_time_metric_infos = setup_generator.get_build_time_metric_infos()
```

Comment:
Indeed, I think that either the build time metric infos should be preserved in the setup directly, or these should be computable using what is stored inside the checkpoint.

Reply:
saved in builder's state